.travis.yml

```diff
@@ -81,10 +85,10 @@ matrix:
           env: CONDA_DEPENDENCIES="`echo $CONDA_DEPENDENCIES | sed 's/ h5py//'`" # this magic incantation removes the substring " h5py" from the dependencies
 
     allow_failures:
-        # The build with numpy v1.11.1rc1 halts in the middle without
-        # showing any obvious reason, thus allowing it to fail for now.
-        - python: 3.5
-          env: NUMPY_VERSION=prerelease SETUP_CMD='test'
+        # Python 3.6 currently has a DeprecationWarning related to pytest
+        # that we don't understand at the moment.
+        - python: 3.6
```

Comment:
you need to repeat the matrix entry both in the main matrix part and using the exact setup under ``allow_failures``.